### PR TITLE
NEW: Angular Date Picker

### PR DESCRIPTION
**Pull request by middle8media**
This pull request makes the following changes:
- Adds https://github.com/alongubkin/angular-datepicker

Test these changes by:
- click "Filters" in toolbar
- scroll to date fields
- click "start date" field
- date picker should slide from bottom
- interact with date picker (pick a date, clear, close etc...)
- repeat for "end date"

Fixes ushahidi/platform#1078

Ping @ushahidi/platform

---
This change is [Reviewable](https://reviewable.io/reviews/ushahidi/platform-client/205)

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ require('angular-resource');
 require('angular-translate');
 require('angular-translate-loader-static-files');
 require('angular-ui-bootstrap');
-require('angular-datepicker');
+require('angular-datepicker/build/angular-datepicker');
```
Comment (rjmackay) [inline app/app.js]: Try this as just `require(‘angular-datepicker’)` - then it loads whatever is set as `main` in package.json for that package

Comment (middle8media) [inline app/app.js]: This breaks the app. I can only get it to work with `require('angular-datepicker/build/angular-datepicker');`

image

Comment (rjmackay) [inline app/app.js]: Yup - you were right to start with. They're package isn't set up properly

**Comment by rjmackay** [comment]
Added this to the post filters and it works perfectly!

**Pull request merged**